openaps_reports.py

```diff
@@ -98,3 +105,12 @@ def recent_dose(self):
                 return self._read_json(Settings.SET_DOSE, [])
 
         return []
+
+    def data_age(self):
+        fn = self._get_report_path(Settings.PREDICT_GLUCOSE)
```

Comment:
ah, ok, I didn't realize. [Here's my loop script](https://github.com/loudnate/rpi-naterade/blob/master/root/every-five.sh), which doesn't continue if any non-zero exit occurs. Could you go that route?

If not, maybe we put the report name in settings to use. If not set, it will use the latest file.

Note that lock files aren't an issue because only files with a `.json` extension appear in the file list, sorted descending by mtime. INI files are always concatenated at the end of that list.